Client/src/main/java/org/xdi/oxauth/client/GluuConfigurationRequest.java

```diff
@@ -0,0 +1,12 @@
+package org.xdi.oxauth.client;
+
+/**
+ * Created by eugeniuparvan on 8/12/16.
+ */
+public class GluuConfigurationRequest extends BaseRequest {
+
+    @Override
+    public String getQueryString() {
+        return null;
```

Comment:
In fact there is no special meaning in this method. The method exists only because his parent (abstract class) forces us to implement it. Since almost all clients in org.xdi.oxauth.client.* are extended from BaseClient<T extends BaseRequest, V extends BaseResponse>, I thought that it would be a good idea if GluuConfigurationClient will also extend from BaseClient, that has led to creation GluuConfigurationRequest and his empty method.